### ISSUE TITLE
Adding canvas_item screen shader to project autoload causes editor crash

### ISSUE THREAD

**Issue by dodgyville** [godotengine/godot#19114]
**Godot version:**
3.x (master)

**OS/device including version:**
Ubuntu 18.04, Nvidia

**Issue description:**
To allow player to adjust game brightness, etc, I have created a simple shader attached to a fullscreen ColorRect under a Control. It works when added to a scene. Rather than add it to every scene in my project I thought I'd push my luck and add it in as an autoload via Project->Project Settings->AutoLoad.

Pressing "add" for the tscn causes editor to crash.

Here's the shader:
```
shader_type canvas_item;

uniform float brightness : hint_brightness;
uniform float contrast : hint_contrast;
uniform float saturation : hint_saturation;

void fragment() {
        vec3 c = texture(SCREEN_TEXTURE, SCREEN_UV).rgb;    

        c.rgb = mix(vec3(0.0), c.rgb, brightness);
        c.rgb = mix(vec3(0.5), c.rgb, contrast);
        c.rgb = mix(vec3(dot(vec3(1.0), c.rgb)*0.33333), c.rgb, saturation);
        COLOR.rgb = c;
}

```

Here's the trace:
```
Dumping the backtrace. Please include this when reporting the bug on https://github.com/godotengine/godot/issues
[1] /lib/x86_64-linux-gnu/libc.so.6(+0x3ef20) [0x7ff8a05aff20] (??:0)
[2] Vector<RasterizerStorageGLES3::RenderTarget::Effects::MipMaps::Size>::operator[](int) (??:0)
[3] RasterizerCanvasGLES3::_copy_texscreen(Rect2 const&) (??:0)
[4] RasterizerCanvasGLES3::canvas_render_items(RasterizerCanvas::Item*, int, Color const&, RasterizerCanvas::Light*, Transform2D const&) (??:0)
[5] VisualServerCanvas::render_canvas(VisualServerCanvas::Canvas*, Transform2D const&, RasterizerCanvas::Light*, RasterizerCanvas::Light*, Rect2 const&) (??:0)
[6] VisualServerViewport::_draw_viewport(VisualServerViewport::Viewport*, ARVRInterface::Eyes) (??:0)
[7] VisualServerViewport::draw_viewports() (??:0)
[8] VisualServerRaster::draw(bool) (??:0)
[9] VisualServerWrapMT::draw(bool) (??:0)
[10] Main::iteration() (??:0)
[11] OS_X11::run() (??:0)
[12] /path_to_project/godot/bin/godot.x11.tools.64(main+0xd5) [0x55689838928f] (??:0)
[13] /lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0xe7) [0x7ff8a0592b97] (??:0)
[14] /path_to_project/godot/bin/godot.x11.tools.64(_start+0x2a) [0x5568983890da] (??:0)
-- END OF BACKTRACE --
```

**Comment by vnen** [comment]
Might be a consequence of #18956.

**Comment by vnen** [comment]
I assume this was fixed by #19108. Comment if you can still reproduce in the latest master.

**Comment by dodgyville** [comment]
Yep, looks fixed. Thanks!
